Description: Don't include sys/pipe.h on kFreeBSD
 Defining _KERNEL to include sys/pipe.h subsequently breaks sys/conf.h;
 since sys/pipe.h is not meant to be used in userland, and libgtop2 builds
 anyway on kFreeBSD without it, don't include it.
Author: Steven Chamberlain <steven@pyro.eu.org>
Bug-Debian: http://bugs.debian.org/667984

Index: libgtop2-2.28.4/sysdeps/freebsd/procmap.c
===================================================================
--- libgtop2-2.28.4.orig/sysdeps/freebsd/procmap.c	2012-04-07 22:49:20.000000000 +0100
+++ libgtop2-2.28.4/sysdeps/freebsd/procmap.c	2012-04-07 22:51:00.163079809 +0100
@@ -38,10 +38,7 @@
 #include <sys/vnode.h>
 #undef _KVM_VNODE
 
-#define _KERNEL
-#include <sys/pipe.h>
 #include <sys/conf.h>
-#undef _KERNEL
 #if (__FreeBSD_version >= 800038) || (__FreeBSD_kernel_version >= 800038)
 #define _WANT_FILE
 #include <sys/file.h>
